forta-bots/Bridge-Invariant/src/L2.DAI.monitor.ts

```diff
@@ -23,16 +24,11 @@ export const provideHandleBlock = (provider: ethers.providers.JsonRpcProvider, d
           metadata: {
             supply: currentSupply.toString(),
           },
-          protocol: "forta-bots-info",
+          protocol: "MakerDAO",
```

Comment:
I think I would keep this as `forta-bots-info`, the idea was to not flood the search by protocols with a bunch of informative alerts. Maybe `forta-bots-info: MakerDAO` is a good one, wdyt?

Reply:
Oh yes, this makes sense 👍